jira/client.py

```diff
@@ -320,6 +320,9 @@ def __init__(self, server=None, options=None, basic_auth=None, oauth=None, jwt=N
                 for name in f['clauseNames']:
                     self._fields[name] = f['id']
 
+        self.field_map = {}
```

Comment:
Can do, just seemed dissimilar to the way the other fields are generated in the rest of the constructor.